charms/mlflow-server/tox.ini

```diff
@@ -54,4 +54,11 @@ commands =
 
 [testenv:unit]
 commands = 
-	pytest tests
+	pytest tests --ignore={[vars]tst_path}integration
+
+[testenv:integration]
+deps = 
+    pytest-operator
+    lightkube
+    requests
+commands = pytest -v --tb native --asyncio-mode=auto --ignore={[vars]tst_path}unit --log-cli-level=INFO -s {posargs}
```

Comment:
Same comment here.  Let's say "integration tests are X" and not "integration tests are everything except Y"